### PR TITLE
move generic swing/awt util classes to common swing package

### PR DESCRIPTION
**Pull request by AndreasEhret**
moved swing/awt util classes to common swing package

### CODE REVIEW

File: common/src/main/java/net/sourceforge/swing/ScreenFinder.java [hunk 1]
```diff
@@ -34,10 +34,9 @@
 this exception to your version of the library, but you are not
 obligated to do so.  If you do not wish to do so, delete this
 exception statement from your version. */
-package net.sourceforge.jnlp.util;
+package net.sourceforge.swing;
```
Comment (hendrikebbers) [inline common/src/main/java/net/sourceforge/swing/ScreenFinder.java]: Maybe we can directly move it in net.adoptopenjdk.... packages?

File: common/src/main/java/net/sourceforge/swing/ThreadCheckingRepaintManager.java [hunk 2]
```diff
@@ -62,29 +60,29 @@ public boolean isCompleteCheck() {
...
     @Override
-    public synchronized void addInvalidComponent(JComponent jComponent) {
+    public synchronized void addInvalidComponent(final JComponent jComponent) {
         checkThreadViolations(jComponent);
         super.addInvalidComponent(jComponent);
     }
 
     @Override
-    public synchronized void addDirtyRegion(JComponent jComponent, int i, int i1, int i2, int i3) {
+    public synchronized void addDirtyRegion(final JComponent jComponent, final int i, final int i1, final int i2, final int i3) {
```
Comment (hendrikebbers) [inline common/src/main/java/net/sourceforge/swing/ThreadCheckingRepaintManager.java]: can we rename the parameters i1, i2,... ?

File: common/src/main/java/net/sourceforge/swing/ScreenFinder.java (hunk 1 above)
Comment (AndreasEhret) [reply common/src/main/java/net/sourceforge/swing/ScreenFinder.java]: done

File: common/src/main/java/net/sourceforge/swing/ThreadCheckingRepaintManager.java (hunk 2 above)
Comment (AndreasEhret) [reply common/src/main/java/net/sourceforge/swing/ThreadCheckingRepaintManager.java]: done

**Comment by hendrikebbers** [comment]
@AndreasEhret can you please rebase :)

**Pull request merged**